base/PerfRunner.php

```diff
@@ -93,6 +93,15 @@ private static function RunWithOptionsAndEngine(
     invariant(!$siege->isRunning(), 'Siege is still running :/');
     invariant($php_engine->isRunning(), get_class($php_engine).' crashed');
 
+    self::PrintProgress('Starting Siege for multi request warmup');
+    $siege = new Siege($options, $target, RequestModes::BENCHMARK);
+    $siege->start();
+    invariant($siege->isRunning(), 'Failed to start siege');
+    $siege->wait();
```

Comment:
other than the request mode, I'm fine with this block existing, but I think an explanation is needed both here and in the readme (or here referencing the readme) saying why there's two warmup modes now.